lib/core/src/sdk.rs

```diff
@@ -2490,12 +2493,19 @@ impl LiquidSdk {
             }
         }
 
+        let liquid_tip = self.liquid_chain_service.lock().await.tip().await?.height;
+        let bitcoin_tip = self.bitcoin_chain_service.lock().await.tip()?.height as u32;
```

Comment:
Are these initiating network requests or the values are taken from a cache? Asking because we already have a block tracker that follows the blocks, perhaps we should also update the blockchain_details there? In a separate key in our cache?
Then we can grab these details directly from there when the user request getinfo and merge it to the response?

Reply:
Yes, those are initiating network requests, but I see what you mean. I'll do as you said and move it into a separate field which we then join.

Reply:
I've updated the `GetInfoResponse` structure (1db033b02d5c), I think now it reflects what we want to show the user. 

I was previously trying to leave the wallet info as-is but was unsatisfied with the workarounds I had to do on the db side to make it work.